sqlsorcery.py

```diff
@@ -0,0 +1,100 @@
+from os import getenv
+from google.oauth2 import service_account
+import pandas as pd
+import pandas_gbq
+from sqlalchemy import create_engine, Table, MetaData
+from sqlalchemy.engine.url import URL
+from sqlalchemy.exc import NoSuchTableError
+from sqlalchemy.schema import DropTable
+
+
+class Connection:
```

Comment:
Can we add docstrings for the class and every public method?

Reply:
Done.